### DISCUSSION TITLE
Next Release?

### DISCUSSION THREAD

**Discussion by aamershahzad2** [stephanstapel/ZUGFeRD-csharp#157] [Q&A] [answered]
Wishing you a great day.
My question is about the Next release.
When will you plan to release the new version of ZUGFeRD-csharp? 
I'm actually using the **ZUGFeRD-csharp 6.0 nuget** and need new attributes for TradeLineItem. For example, **ApplicableProductCharacteristic**.

I appreciate your help.

**Comment by stephanstapel** [comment, marked as answer]
good point. I'll do that in the next days.

**Reply by EugenFink13** [reply]
Hi Stephan,
do you plan to make a release in next view days?
We need BillingPeriod, DRV requires it. 

I appreciate your help.

**Reply by stephanstapel** [reply]
done: 
https://www.nuget.org/packages/ZUGFeRD-csharp/

Still looking for sponsoring:
https://github.com/sponsors/stephanstapel

**Comment by AlexanderEichhorn** [comment]
Hi Stephan, what about a new NuGet Version/Release? Thanks for this great Project!

Best Regards

**Comment by stephanstapel** [comment]
Hi,

this is done:
https://github.com/stephanstapel/ZUGFeRD-csharp/releases/tag/8.0.0

If you like the project, I'd be more than happy if you could support me:
https://github.com/sponsors/stephanstapel
